third_party/nvidia/backend/compiler.py

```diff
@@ -322,7 +316,8 @@ def make_ttgir(mod, metadata, opt, capability):
         passes.common.add_canonicalizer(pm)
 
         pm.run(mod, 'make_ttgir')
-        metadata["cluster_dims"] = (cluster_info.clusterDimX, cluster_info.clusterDimY, cluster_info.clusterDimZ)
+        # num_ctas == 16 is non-portable. Does work for H100 and B200 tho
+        metadata["cluster_dims"] = (opt.num_ctas, 1, 1)
```

Comment:
NIT: this comment doesn't really make sense here. It would make more sense in the driver template.